lib/logger/lib/logger.ex

```diff
@@ -304,7 +304,8 @@ defmodule Logger do
   end
 
   @doc """
-  Adds the given keyword list to the current process metadata.
+  Adds the given keyword list to the current process metadata. This also allows you to 
```

Comment:
I think we can expand how it works in details in second paragraph:
```
Alters the current process metadata according the given keyword list.

<describes what happens in case of `nil` value under a key>
```